.github/workflows/supported_jdks.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
         jdk:
           - '25-ea'
 
+    name: ${{ matrix.jdk == '25-ea' && 'Build Early Access' }}
```

Comment:
Isn't this effectively the same as removing the matrix given that we only have one entry?